packages/gallery/src/components/item/itemView.js

```diff
@@ -294,8 +290,7 @@ class ItemView extends React.Component {
         hoveringBehaviour !==
           GALLERY_CONSTS[
             optionsMap.behaviourParams.item.overlay.hoveringBehaviour
-          ].NEVER_SHOW &&
-        (allowDescription || allowTitle || isStoreGallery)
+          ].NEVER_SHOW
       ) {
```

Comment:
@shmuelhizmi I removed allowDescription || allowTitle || isStoreGallery params from the condition, they aren't related to the core